build.gradle.kts

```diff
@@ -166,7 +166,7 @@ buildTypes {
     }
 
     create("soakTest") {
-        tasks("soak:soakTest")
+        tasks("soak:soakIntegTest")
```

Comment:
I'm not too happy about this. We also have `smokeTest` which is not an `Integ` test... That said, I'm okay with leaving it like it is for now.

Reply:
Better ideas? I changed this so it doesn't clash with the build type of the same name and I can run the test from IDEA.

Reply:
No better ideas.